index.js

```diff
@@ -1,80 +1,93 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] It would be better if you extract your classes into multiple files so that you can remove eslint disable and also will make your project organized and clean

Reply:
We extracted the classes into multiple JS files.